packages/camera/src/components/Capture/capture.js

```diff
@@ -415,7 +428,8 @@ const Capture = forwardRef(({
       onFinishUploadPicture={onFinishUploadPicture}
       addDamageParts={addDamageParts}
       onResetAddDamageStatus={handleResetDamageStatus}
-      hideAddDamage={hideAddDamage}
+      hideAddDamage={hideAddDamage || isPortraitModeVinLayoutView }
```

Comment:
Make a change in hideAddDamage memo with isPortraitModeVinLayoutView instead of using it here...